### PR TITLE
Adding support for setting color chroma and color rgb values in the r…

### PR DESCRIPTION
**Pull request by ChuckFerring-zz**
…eference app.

### CODE REVIEW

File: reference-app/app.js [hunk 1]
```diff
@@ -180,6 +180,19 @@ mainModule.controller('MainCtrl', ['$scope', '$http', '$q', 'remote', 'config',
...
     }
 
+    $scope.getResourceStyle = function(resource) {
+        let style = {};
+
+        if(resource.rt[0] === 'oic.r.colour.rgb') {
+            let rgb = resource.rgbvalue;
+            if(rgb && rgb[0] !== null && rgb[1] !== null && rgb[2] !== null) {
+                style['background-color'] = `rgb(${Math.round(rgb[0])},${Math.round(rgb[1])},${Math.round(rgb[2])})`;
+            }
```
Comment (shalinijoshi19) [inline reference-app/app.js:190]: else what happens ? error logged somewhere? is this even a valid scenario where these values will be null? if not should we warn?

**Comment by shalinijoshi19** [comment]
:shipit:

**Review by shalinijoshi19 — APPROVED** [review]

File: reference-app/app.js [hunk 2]
```diff
@@ -180,6 +180,19 @@ mainModule.controller('MainCtrl', ['$scope', '$http', '$q', 'remote', 'config',
         return true;
     }
 
+    $scope.getResourceStyle = function(resource) {
+        let style = {};
+
+        if(resource.rt[0] === 'oic.r.colour.rgb') {
```
Comment (arjun-msft) [inline reference-app/app.js:186]: >'oic.r.colour.rgb' [](start = 30, length = 18)

what about chroma?

**Comment by arjun-msft** [comment]
:shipit:

**Review by danielrischar — APPROVED** [review]

**Pull request merged**